src/components/Sing/SequencerNote.vue

```diff
@@ -99,33 +82,19 @@ import ContextMenu, {
   ContextMenuItemData,
 } from "@/components/Menu/ContextMenu.vue";
 
-type NoteState = "NORMAL" | "SELECTED";
-
-const vFocus = {
-  mounted(el: HTMLInputElement) {
-    el.focus();
-    el.select();
-  },
-};
-
-const props = withDefaults(
-  defineProps<{
-    note: Note;
-    isSelected: boolean;
-    previewLyric: string | null;
-  }>(),
-  {
-    isSelected: false,
-  },
-);
+const props = defineProps<{
+  note: Note;
+  nowPreviewing: boolean;
+  isSelected: boolean;
+  isPreview: boolean;
```

Comment:
nowPreviewingとisPreviewがややこしいかもですね！
説明するならこんな感じでしょうか。
```suggestion
  /** どれかのノートがプレビュー中 */
  nowPreviewing: boolean;
  /** このノートが選択中か */
  isSelected: boolean;
  /** このノートがプレビュー中か */
  isPreview: boolean;
```